exporter/exporterhelper/internal/bounded_memory_queue_test.go

```diff
@@ -107,13 +107,15 @@ func TestBoundedQueue(t *testing.T) {
 // only after Stop will mean the consumers are still locked while
 // trying to perform the final consumptions.
 func TestShutdownWhileNotEmpty(t *testing.T) {
-	q := NewBoundedMemoryQueue(10, 1)
+	q := NewBoundedMemoryQueue(1000, 5)
 
 	consumerState := newConsumerState(t)
 
+	var wg sync.WaitGroup
+	wg.Add(1)
```

Comment:
For a single producer/consumer blocking code, use a channel, requires 1 less line :)

Reply:
👍